facades/PC/build.gradle

```diff
@@ -86,6 +109,17 @@ configurations {
     }
 }
 
+run {
```

Comment:
I'm guessing this is new with the `application` plugin, but if so shouldn't it replace the `game` task? Or maybe they can be used in different cases with one doing something more advanced than another

Also as a reminder there's a new exec task in here, so might want to rebase to grab it :-)

Reply:
yep, it's a task provided by `application`. and yes, the game* tasks could probably be based off that instead and probably save some of the default configuration. This part is sitting on an awkward "what is in scope for this PR?" line, as I didn't want to get in to all the existing JavaExec tasks.

there's a case to be made that I should leave that `run` addition out of this PR and leave it for that day.

Reply:
Okay no worries then, keep it simple and we'll just continue to iterate :-)